docs/source/references/config.md

```diff
@@ -773,6 +774,7 @@ Has type `WorldStateViewConfiguration`. Can be configured via environment variab
   },
   "WASM_RUNTIME_CONFIG": {
     "FUEL_LIMIT": 1000000,
+    "IROHA_WASM_MODULE": "/iroha_core_wasm.wasm",
```

Comment:
In this design, the WASM module that is being used can be hot-plugged. While I like the modularity, I think that this can come back to haunt us later. 

For example, how do we handle the modules' security patches. How do we ensure that the upgrade got propagated to all of the Iroha instances, without administrator intervention. What if the original WASM binary was located in a read-only file-system and is no-longer accessible. 

To me it seems like a better course of action would be to hard-code the WASM binary into Iroha itself. As such it should be impossible to update just the WASM, and we don't have a multitude of semver incompatible APIs, just one.

Reply:
I'd be interested to hear what @SamHSmith and @s8sato think about this.

Reply:
Since this design has the Wasm binary of the intermediary module on the Iroha side,
I think we can use API versioning for consistency between the two Wasm modules